README.md

```diff
@@ -29,3 +29,10 @@ The installer scripts also work fine without docker; just run the following two
 
 The unpacking requires recent versions of msitools (0.98) and libgcab
 (1.2); sufficiently new versions are available in e.g. Ubuntu 19.04.
+
+
+# Build instructions for native environments
+
+See [INSTRUCTIONS.native.md](INSTRUCTIONS.native).
```

Comment:
I'd prefer to have all of this included in the main README.md, instead of splitting out a separate file. I agree that the docker setup isn't what most people might be using in the end, but it's great as a sealed, self contained and reproducible example for how it works. So the main README could be 1. minimal docker example, 2. normal local installation examples 3. FAQ.

Also as name for this kind of installation, I'd prefer "local installation" or something else to denote the non-docker installation setup, rather than "for native environments".

Reply:
I wasn't exactly sure how to handle it.

The way I see there are two very distinct users who want very different things:

1. CI devs who want a container that "just works" to test MSVC projects still work on Windows without having to launch a Windows server.
   - A slight variation of those who want a CI for do stuff like godbolt (i.e. online compiler)
2. Devs who want to compile in a native environment. Often for iterating (i.e. build -> fix errors -> build -> launch -> fix errors -> build); which means tinkering with the compiler until it works with their IDE (btw it works with QtCreator! Although Code Autocomplete & Syntax highlighting breaks because Clang plugin doesn't notice it's an MSVC installation. I alternate between mingw64 for syntax highlighting, then to MSVC for building. It's good enough for me for now)

Reply:
Sure, but even if there are these two cases, I'd want this info to be available in the main readme, not hidden away in a separate document. So please merge it into the main readme, with the ordering I suggested (first simple docker example, then regular local install, then FAQ).